### DISCUSSION TITLE
Is bpf2go related to the version of libbpf?

### DISCUSSION THREAD

**Discussion by binw666** [cilium/ebpf#844] [Q&A] [answered]
I found bpf_map_def has been removed from libbpf/src/bpf_helpers.h in v1.0.1. Is bpf2go just a compiler? Can it work with any version of libbpf?

**Comment by brycekahle** [comment, marked as answer]
You wouldn't use both `bpf2go` and `libbpf`. `bpf2go` is designed to work with this library, which is an eBPF loader. `libbpf` is also a loader, and has a feature somewhat like `bpf2go` called skeletons.

**Reply by binw666** [reply]
OK. I get it. Thank you very much!
